README.md

```diff
@@ -134,7 +134,7 @@ var colorCursor = paletteCursor.select('colors');
 
 #### Updates
 
-A *baobab* tree can obviously be updated. However, one has to understand that he won't do it, at least by default, synchronously.
+A *baobab* tree can obviously be updated. However, one has to understand that they won't do it, at least by default, synchronously.
```

Comment:
`he or she` would accomplish the same goal, but I prefer the singular `they`, myself.